source/com.google.mlkit/common/Transforms/Metadata.Namespaces.xml

```diff
@@ -0,0 +1,54 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<metadata>
+
+    <attr
+        path="/api/package[@name='com.google.android.gms.internal.mlkit_common']"
+        name="managedName"
+        >
+        Android.Gms.Internal.MLKitCommon
```

Comment:
Should this be `Android.Gms.Internal.MLKit.Common` for consistency with `Xamarin.GoogleFirebase.MLKit.Common`?

Reply:
Not sure how to handle `_`.
Note: it is not MLKit, but GMS namespace (GooglePlayServices - old name Google Messaging Services)

I tend not to use namespaces which are defined somewhere else. In this case `MLKit` in `MLKit.Common` vs `MLKitCommon. It is similar to avoiding `Android`, so I tend to use `XamarinAndroid`. Less problems when resolving namespaces.